packages/insomnia-inso/src/commands/lint-specification.js

```diff
@@ -15,13 +18,33 @@ export async function lintSpecification(
 
   const specFromDb = identifier ? loadApiSpec(db, identifier) : await promptApiSpec(db, !!ci);
 
-  if (!specFromDb) {
-    logger.fatal(`Specification not found.`);
+  let specContent: string = '';
+
+  try {
+    if (specFromDb?.contents) {
+      logger.trace('Linting specification from database contents');
+      specContent = specFromDb.contents;
+    } else if (identifier) {
+      // try load as a file
+      const fileName = path.isAbsolute(identifier)
+        ? identifier
+        : path.join(workingDir || '.', identifier);
+      logger.trace(`Reading specification from file \`${fileName}\``);
```

Comment:
Please update this trace to the following

```suggestion
      logger.trace(`Linting specification from file \`${fileName}\``);
```